x/nameservice/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,269 @@
+package keeper_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/tharsis/ethermint/x/nameservice/client/cli"
+	nameservicetypes "github.com/tharsis/ethermint/x/nameservice/types"
+	"os"
+)
+
+func (suite *KeeperTestSuite) TestGrpcQueryParams() {
+	grpcClient := suite.queryClient
+
+	testCases := []struct {
+		msg string
+		req *nameservicetypes.QueryParamsRequest
+	}{
+		{
+			"Get Params",
+			&nameservicetypes.QueryParamsRequest{},
+		},
+	}
+	for _, test := range testCases {
```

Comment:
Let's avoid tabular tests if we have just one test case

Reply:
Okay